packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -45,13 +46,8 @@ class DateAbstract extends React.Component {
   }
 
   handleOnChange(selectedDates) {
-    if (!selectedDates || selectedDates.length === 0) {
-      this.props.onChange(null)
-    } else if (selectedDates.length === 1) {
-      this.props.onChange(selectedDates[0].toISOString())
-    } else {
-      throw new Error('EditableValue DateTime: unexpected length of input')
-    }
+    const asIsoString = selectedDates.map(date => date.toISOString())
```

Comment:
Just a detail but maybe name asIsoString = isoStrings? if you call it as.. i expect a function